### PR TITLE
gae-interop-testing: kokoro script to run interop tests in GAE

### PR DESCRIPTION
**Pull request by zpencer**
GAE interop tests:
- add runtime configs to jdk7 and jdk8 projects 

kokoro scripts: 
- make sure a 'dummy-default' version always exists
- when the tests are done, divert all traffic to dummy-default and delete the version deployed for tests

### CODE REVIEW

**Comment by zpencer** [comment]
@jtattermusch @matt-kwong the script is stable now, PTAL

File: buildscripts/kokoro/gae-interop.sh [hunk 1]
```diff
@@ -1,3 +1,75 @@
-#!/bin/sh
+#!/bin/bash
 
-# An empty placeholder shellscript
+set -exu -o pipefail
+
+pwd
+hostname
```
Comment (matt-kwong) [inline buildscripts/kokoro/gae-interop.sh]: It looks like these are leftover commands from exploring Kokoro. These should be removed.

Comment (zpencer) [reply buildscripts/kokoro/gae-interop.sh]: Fixed

**Comment by zpencer** [comment]
The test servlet running in GAE aggregates all the failed test case names and exceptions and puts it in the HTTP response body. The gradle script prints the `result?.body()?.string()` so we print that in the test log. It still has the problem of external users not being able to read them by default, but it does not require that people have access to the GAE logs.

**Comment by zpencer** [comment]
Hi @jtattermusch , can I get an approval on this if there are no issues?

**Pull request merged**